include/itkBioCellularAggregate.hxx

```diff
@@ -36,7 +36,7 @@ CellularAggregate< NSpaceDimension >
 
   m_Mesh = MeshType::New();
 
-  m_Mesh->SetCellsAllocationMethod(MeshType::CellsAllocatedDynamicallyCellByCell);
+  m_Mesh->SetCellsAllocationMethod(itk::MeshClassCellsAllocationMethodType::CellsAllocatedDynamicallyCellByCell);
```

Comment:
> /ITKBioCell/include/itkBioCellularAggregate.hxx:39:41: error: 'itk::MeshClassCellsAllocationMethodType' has not been declared

Maybe you also need outermost scope `::`:
```C++
m_Mesh->SetCellsAllocationMethod(::itk::MeshClassCellsAllocationMethodType::CellsAllocatedDynamicallyCellByCell);
```

Reply:
This unfortunately did not seem to do the trick, I will continue to try to fix the error.